### ISSUE TITLE
KEMIKL

### ISSUE THREAD

**Issue by johnbent** [tekinged/missing#482]
Hidden?

**Comment by Gnalu** [comment]
Or overlooked purposely.

**Comment by Lukes-Imeyuns** [comment]
Yes, it refers to when someone is ignored, left in the periphery, excluded
by others, purposely or not by others.

So when I finally see a friend or an aunt after a long time they might say
"Ke kora di mechesang ra urerem me ar bebil er a rechad me a kora di ngara
kemikr kung."

Or someone who is never or only rarely invited to family gatherings or
included in meetings and such might say "Ak blechoel di ngara kemikr."

This makes me think of "chopped liver" Lol
